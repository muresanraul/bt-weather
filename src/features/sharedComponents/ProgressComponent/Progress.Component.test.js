import React from 'react';
import renderer from 'react-test-renderer';

import { default as ProgressComponent } from './Progress.Component';

describe('ProgressComponent:', () => {

  it('renders width the progress correctly - 0 sec', () => {
    const progressComponent = renderer.create(
      <ProgressComponent time={0} />
    ).toJSON();
    expect(progressComponent).toMatchSnapshot();
  });

  it('renders width the progress correctly - 25 sec', () => {
    const progressComponent = renderer.create(
      <ProgressComponent time={25} />
    ).toJSON();
    expect(progressComponent).toMatchSnapshot();
  });

  it('renders width the progress correctly - 60 sec', () => {
    const progressComponent = renderer.create(
      <ProgressComponent time={60} />
    ).toJSON();
    expect(progressComponent).toMatchSnapshot();
  });

});
