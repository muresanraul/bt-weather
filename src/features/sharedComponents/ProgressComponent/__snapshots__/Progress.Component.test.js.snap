// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressComponent: renders width the progress correctly - 0 sec 1`] = `
<div
  className="progress"
>
  <div
    className="progress__bar"
    style={
      Object {
        "width": "0%",
      }
    }
  />
</div>
`;

exports[`ProgressComponent: renders width the progress correctly - 25 sec 1`] = `
<div
  className="progress"
>
  <div
    className="progress__bar"
    style={
      Object {
        "width": "42%",
      }
    }
  />
</div>
`;

exports[`ProgressComponent: renders width the progress correctly - 60 sec 1`] = `
<div
  className="progress"
>
  <div
    className="progress__bar"
    style={
      Object {
        "width": "100%",
      }
    }
  />
</div>
`;
